python/interpret_community/widget/explanation_dashboard.py

```diff
@@ -193,15 +243,13 @@ def __init__(self, explanation, model=None, *, dataset=None,
             local_url = "{0}/{1}".format(
                 base_url,
                 str(ExplanationDashboard.model_count))
-        explanation_input =\
-            ExplanationDashboardInput(explanation, model, dataset, true_y, classes, features, predict_url, locale)
+        with_credentials = ExplanationDashboard.service.with_credentials
+        explanation_input = ExplanationDashboardInput(explanation, model, dataset, true_y, classes,
+                                                      features, predict_url, locale, with_credentials)
         # Due to auth, predict is only available in separate tab in cloud after login
-        if ExplanationDashboard.service.env == "local":
+        if ExplanationDashboard.service.env != "cloud":
             explanation_input.enable_predict_url()
         html = generate_inline_html(explanation_input, local_url)
-        if ExplanationDashboard.service.env == "azure":
```

Comment:
if this now works in azure environments, the comment on line 249 can be removed :)